main/xen/APKBUILD

```diff
@@ -107,7 +107,7 @@ options="!strip"
 #     - XSA-250
 #     - XSA-251
 #   4.9.1-r3:
-#     - XSA-254 XPTI
+#     - XSA-254 XPTI "stage 1" (CVE-2017-5753,CVE-2017-5715,CVE-2017-5754)
```

Comment:
Slightly changed the format of this entry in order to match the standard:

- CVE entries first, space separated
- Parenthesis and comment in bracket removed